api/v1/testkube.yaml

```diff
@@ -649,6 +718,129 @@ paths:
 
 components:
   schemas:
+
+    ObjectRef:
+      type: object
+      properties:
+        namespace:
+          type: string
+        name:
+          type: string
+
+
+    Test:
+      type: object
+      properties:
```

Comment:
perhaps we should have a description property - testers like to describe what their tests do...